mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -280,3 +280,89 @@ SONIC:
 		Offset: -170,0,0
 	AutoTarget:
 	WithVoxelTurret:
+
+JUGG:
+	Inherits: ^Tank
+	Inherits@SPRITES: ^SpriteActor
+	Inherits@EXPERIENCE: ^GainsExperience
+	Valued:
+		Cost: 950
+	Tooltip:
+		Name: Juggernaut
+		RequiresCondition: !deployed
+	Tooltip@DEPLOYED:
+		Name: Juggernaut (deployed)
+		RequiresCondition: deployed
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 100
+		Prerequisites: ~gaweap, garadr, ~techlevel.high
+		Description: Mobile Artillery Mech.\nNeeds to deploy in order to shoot.\n  Strong vs Ground units\n  Weak vs Aircraft
+	Health:
+		HP: 350
+	Armor:
+		Type: Light
+	Mobile:
+		Speed: 71
+		TurnSpeed: 5
+		RequiresCondition: !empdisable && undeployed
+	RevealsShroud:
+		RequiresCondition: !inside-tunnel
+		Range: 9c0
+		MaxHeightDelta: 3
+	RenderVoxels:
+		LightAmbientColor: 0.4, 0.4, 0.4
+	RenderSprites:
+		Image: jugg
+	WithMakeAnimation:
+		BodyNames: body, deployedbody
+	WithFacingSpriteBody:
+		Sequence: stand
+		RequiresCondition: undeployed
+	WithMoveAnimation:
+		MoveSequence: run
+	GrantConditionOnDeploy:
+		DeployedCondition: deployed
+		UndeployedCondition: undeployed
+		Facing: 96
+		AllowedTerrainTypes: Clear, Road, DirtRoad, Rough
+		DeploySound: place2.aud
+		UndeploySound: clicky1.aud
+	GrantCondition@PREVIEWWORKAROUND:
+		Condition: real-actor
+	QuantizeFacingsFromSequence:
+		Sequence: turret
+	WithSpriteBody@deployed:
+		RequiresCondition: !undeployed && real-actor
+		Name: deployedbody
+	Turreted:
+		Turret: deployed
+		TurnSpeed: 5
+		InitialFacing: 96
+		Offset: -108,-12,448
+		RealignDelay: -1
+	WithVoxelBarrel:
```

Comment:
The original game would automatically rotate the pitch of the barrels to match the firing arc.  Doing this properly is very far down my TODO list, so I have filed https://github.com/reaperrr/OpenRA/pull/15 for you to include here that lets hardcode a more sensible fixed angle than horizontal.